eslint.config.mjs

```diff
@@ -0,0 +1,12 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+
+export default [
+	{
+		languageOptions: { globals: globals.browser },
```

Comment:
I usually see this option at the last position (so it can override previous ones).

I guess giving this configuration it should work :tm: , though.